Views/EditorTemplates/Parts/AdvancedSKUProduct.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿@model Nwazet.Commerce.ViewModels.AdvancedSKUProductEditorViewModel
+
+@Html.HiddenFor(m => m.Product.Sku)
+
+@{
+    string skuHtml = "";
+}
+@if (Model.Settings.AllowCustomPattern) {
```

Comment:
All this concatenation and escaping is hard to follow and probably worse to maintain. I'd prefer to let Razor do its job (maybe in a hidden div), and have the script copy the results.